protocol-units/settlement/mcr/client/abi/MCR.json

```diff
@@ -1 +1 @@
enesisCeremonyEnded","outputs":[{"internalType":"bool","name":"","type":"bool"}]},{"inputs":[],"stateMutability":"view","type":"function","name":"lastAcceptedBlockHeight","outputs":[{"internalType":"uint256","name":"","type":"uint256"}]},{"inputs":[],"stateMutability":"view","type":"function","name":"leadingBlockTolerance","outputs":[{"internalType":"uint256","name":"","type":"uint256"}]},{"inputs":[],"stateMutability":"payable","type":"function","name":"stake"},{"inputs":[],"stateMutability":"payable","type":"function","name":"stakeGenesis"},{"inputs":[{"internalType":"struct MCR.BlockCommitment[]","name":"blockCommitments","type":"tuple[]","components":[{"internalType":"uint256","name":"height","type":"uint256"},{"internalType":"bytes32","name":"commitment","type":"bytes32"},{"internalType":"bytes32","name":"blockId","type":"bytes32"}]}],"stateMutability":"nonpayable","type":"function","name":"submitBatchBlockCommitment"},{"inputs":[{"internalType":"struct MCR.BlockCommitment","name":"blockCommitment","type":"tuple","components":[{"internalType":"uint256","name":"height","type":"uint256"},{"internalType":"bytes32","name":"commitment","type":"bytes32"},{"internalType":"bytes32","name":"blockId","type":"bytes32"}]}],"stateMutability":"nonpayable","type":"function","name":"submitBlockCommitment"},{"inputs":[{"internalType":"uint256","name":"amount","type":"uint256"}],"stateMutability":"nonpayable","type":"function","name":"unstake"}],"devdoc":{"kind":"dev","methods":{},"version":1},"userdoc":{"kind":"user","methods":{},"version":1}},"settings":{"remappings":["@openzeppelin/contracts/=lib/openzeppelin-contracts/contracts/","ds-test/=lib/openzeppelin-contracts/lib/forge-std/lib/ds-test/src/","erc4626-tests/=lib/openzeppelin-contracts/lib/erc4626-tests/","forge-std/=lib/forge-std/src/","murky/=lib/murky/","openzeppelin-contracts/=lib/openzeppelin-contracts/","openzeppelin/=lib/openzeppelin-contracts/"],"optimizer":{"enabled":true,"runs":200},"metadata":{"bytecodeHash":"ipfs"},"compilationTarget":{"src/settlement/MCR.sol":"MCR"},"evmVersion":"paris","libraries":{}},"sources":{"lib/forge-std/src/console.sol":{"keccak256":"0x91d5413c2434ca58fd278b6e1e79fd98d10c83931cc2596a6038eee4daeb34ba","urls":["bzz-raw://91ccea707361e48b9b7a161fe81f496b9932bc471e9c4e4e1e9c283f2453cc70","dweb:/ipfs/QmcB66sZhQ6Kz7MUHcLE78YXRUZxoZnnxZjN6yATsbB2ec"],"license":"MIT"},"lib/openzeppelin-contracts/contracts/utils/structs/EnumerableSet.sol":{"keccak256":"0x86c1470cbfd878491e5de030072b647352d36bd27122cffb928970b1945282aa","urls":["bzz-raw://ad85dd393ee0a1917c57046abc5155f51f77844b2c6a42c05c1b8dd26d6ff3c1","dweb:/ipfs/QmNqYc8To2NdnpP6E1tGz7t6A7beuENde5yovwov5pW1fA"],"license":"MIT"},"src/settlement/MCR.sol":{"keccak256":"0x52ba2c46a7fa9f952ce39afe0ee87dcb063bc6a5cd80a88a04aecb102b98ea95","urls":["bzz-raw://797b2406271ac464e7bf5dff285c973716c25e09e592519d467341acb3bc2c49","dweb:/ipfs/QmTYWSBK3Jcrftp8dXCeT2DoPeMBgFVVN8JoRX1A1qZrBY"],"license":"UNLICENSED"}},"version":1},"id":26}
```

Comment:
This file looks auto-generated, do we need to check it in?

Reply:
The client needs the ABI to build (we've ignored unnecessary ABIs elsewhere).